homeassistant/components/braviatv/config_flow.py

```diff
@@ -160,7 +160,8 @@ def __init__(self, config_entry):
 
     async def async_step_init(self, user_input=None):
         """Manage the options."""
-        self.braviarc = self.hass.data[DOMAIN][self.config_entry.entry_id][BRAVIARC]
+        client = self.hass.data[DOMAIN][self.config_entry.entry_id][BRAVIA_CLIENT]
+        self.braviarc = client.get_braviarc()
```

Comment:
Just access the attribute.
```suggestion
        self.braviarc = coordinator.braviarc
```